### PR TITLE
Replacing existing Legal section with new content

### PR DESCRIPTION
**Pull request by cwarnermm**
Updates based on [Legal project plan](https://docs.google.com/spreadsheets/d/12Bk_hKRB18h-G6q76M5qnJiShb0LlaZgOCRNpMaIX7o/edit#gid=0).

Adding @amyblais as editor for PR approval. Legal to review updated Handbook section in production and provide feedback as needed.

### CODE REVIEW

File: operations/legal/ironclad-basics/requesting-template-in-spotdraft.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Non-Disclosure Agreements
+
+* Whenever possible you should always be using the Mattermost template. A Counterparty may send you their NDA automatically but as the Workflow Owner, you should push back, at least once but preferably twice, to use Mattermost’s template.  If you must use the Counterparty’s paper, please see the section Requesting Review of a Third-Party Contract.
+* You can find the more information about the NDA workflow, including a video [here](https://drive.google.com/drive/folders/1mFc7heoplPEGQ9CwMhoieT701xX0QCCP?usp=sharing).
+
+## Requesting review of a Third-party contract
+
+* Whenever possible you should always be using the applicable Mattermost template. A Counterparty may send you their template but as the Workflow Owner, you should push back, at least once but preferably twice, to use Mattermost’s template.
+* For any **Procurement related contracts**, please submit via ** Airbase** which now includes Legal’s review automatically within Ironclad. For more information please see [How to submit a purchase request](https://handbook.mattermost.com/operations/finance/airbase/how-to-submit-a-purchase-request).
```
Comment (amyblais) [inline operations/legal/ironclad-basics/requesting-template-in-spotdraft.md]:
```suggestion
* For any **Procurement related contracts**, please submit via **Airbase** which now includes Legal’s review automatically within Ironclad. For more information please see [How to submit a purchase request](https://handbook.mattermost.com/operations/finance/airbase/how-to-submit-a-purchase-request).
```

File: operations/legal/legal.md [hunk 2]
```diff
@@ -1,60 +1,62 @@
...
-While this takes slightly longer, it pays-off long term by answering questions once.
+* [Software and Services License Agreement](https://mattermost.com/software-services-license-agreement/)
+* [Data Processing Addendum](https://mattermost.com/data-processing-addendum/)
+* [Privacy Policy](https://mattermost.com/privacy-policy/)
 
-Also, if you see parts of our documentation or handbook out-of-date, please also mail legal@mattermost.com to let us know.
+### US Government Contracts
 
-### Outside council one-pager
+* [Mattermost Legal Terms](https://mattermost.com/terms-of-use/)- Our standard legal terms for using Mattermost software and services.
```
Comment (amyblais) [inline operations/legal/legal.md]:
```suggestion
* [Mattermost Legal Terms](https://mattermost.com/terms-of-use/) - Our standard legal terms for using Mattermost software and services.
```

**Review by amyblais — DISMISSED** [review]
Two minor formatting errors, otherwise looks good:

**Review by amyblais — APPROVED** [review]

**Comment by cwarnermm** [comment]
Thanks, @amyblais!

**Pull request merged**